internal/plugins/ansible/templates/config/rbac/kustomization.go

```diff
@@ -101,10 +101,10 @@ const kustomizeTemplate = `resources:
 - auth_proxy_role.yaml
 - auth_proxy_role_binding.yaml
 - auth_proxy_client_clusterrole.yaml
+patchesJson6902:
```

Comment:
what is patchesJson6902?

Reply:
It's the way to specify a JSON merge patch https://kubectl.docs.kubernetes.io/pages/reference/kustomize.html#patchesjson6902

Reply:
why do we need to do this change? What is the end goal? 
What is the problem that you are trying to solve with?

Reply:
It allows us to add RBAC for CRDs without modifying the central role.yaml that the user configures, and means we don't need an updater for the role.yaml

Reply:
But then you change the common the purposed of this files. They should not be used by us.